lib/resources/payments.js

```diff
@@ -51,7 +51,15 @@ module.exports = function (api) {
       if (!amount) {
         throw new Error('`amount` is mandatory')
       }
-
+      if (callback && typeof callback === 'string') {
```

Comment:
In this case, the fourth argument _might_ be a callback if someone wants to pass currency as well as a callback. The usage in that case would be `capture('pay_id', 100, 'USD', doSomethingAfterCapture)`.

The current implementation will not pass the callback to `api.post` if a callback exists along with currency, it should.